app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
```

Comment:
I think validating length of :name as at least 3 and also setting null: false will do, no need for presence in my opinion